script2.js

```diff
@@ -1,15 +1,15 @@
 $(document).ready(function() {
   $('#addTask').click(function() {
-    let $name = $('#usName').val();
+    let $names = $('#usName').val();
     let $emp = $('#eid').val();
     let $city = $('#c_name').val();
-    let $checks = validate($name, $city, $emp);
```

Comment:
When naming a function, always make sure the name of the function is meaningful. For eg: when you write validate, then it makes a 3rd person think validate what?

Reply:
changed function name validate to inputValidate()